.github/workflows/release-security-data-access-compatibility-53.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         shell: pwsh
     outputs:
       hash-code: ${{ steps.hash-code.outputs.hash-code }}
-      security-data-access-version: 5.3.66
+      security-data-access-version: 5.3.67
```

Comment:
I saw this change in previous commits, do you know if this is the correct value ?

Reply:
Needs to match the version number in https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Compatibility-Libraries/blob/main/NuGet/EdFi.Security.DataAccess/53/Directory.Build.props

Let's call this 5.4.0.